docs/esp32/quickref.rst

```diff
@@ -110,6 +113,29 @@ calling ``wlan.config(reconnects=n)``, where n are the number of desired reconne
 attempts (0 means it won't retry, -1 will restore the default behaviour of trying
 to reconnect forever).
 
+LAN
+^^^
+
+To use the wired interfaces one has to specify the pins ::
+
+    import network
+
+    lan = network.LAN(mdc=PIN_MDC, ...)   # Set the pin configuration
+    lan.active(True)                      # activate the interface
+    lan.ifconfig()                        # get the interface's IP/netmask/gw/DNS addresses
+
+
+These are working configurations for LAN interfaces of popular boards:
+
+Olimex ESP32-GATEWAY::
+
+   network.LAN(mdc=machine.Pin(23), mdio=machine.Pin(18), power=machine.Pin(5), phy_type=network.PHY_LAN8720, phy_addr=0, clock_mode=network.ETH_CLOCK_GPIO17_OUT)
```

Comment:
note that `clock_mode` was removed for IDF 4.x LAN, so these configurations will not work anymore

Reply:
Ethernet didn't work yet for IDF4 when I tried the configurations.

Reply:
did you get a chance to try the new code out now with these configuration lines?